src/platform/forms-system/src/js/definitions/directDeposit.jsx

```diff
@@ -38,12 +38,6 @@ const uiSchema = ({ affectedBenefits, unaffectedBenefits, optionalFields }) => {
         </div>
       ) : null;
     },
-    'ui:order': [
-      'bankAccount',
-      'declineDirectDeposit',
-      'view:directDespositInfo',
-      'view:bankInfoHelpText',
-    ],
```

Comment:
Why is this being removed?

Reply:
![image](https://user-images.githubusercontent.com/1094999/113444363-bcc60f80-93c1-11eb-9051-845735b9d6da.png)
on the review page of 1990s i was receiving this error message regarding the view properties in both `ui:order`

Reply:
Change was done to schema creation to ensure order is correct

Reply:
I attempted to remove the properties from `ui:order` which allowed for the Review page to render but did allow for the page in the form to render.

I tried using as well
```
'ui:options': {
        hideOnReview: true,
      },
```

Reply:
Based on that screenshot it looks like you only needed to remove the properties starting with `view:`.